deployment/src/main/resources/values.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+replicaCount: 1
+
+image:
+  name: @artifactId@
+  tag: @revision@-@git.commit.id.abbrev@
+  pullPolicy: IfNotPresent
+
+resources:
+  limits:
+    cpu: 750m
+    memory: 2.5Gi
+  requests:
+    cpu: 250m
+    memory: 1Gi
```

Comment:
Ditto, bring these down

Reply:
(this is the file to change, then a -P pi will update the other)

Reply:
Changed and done `-P pi`